trento/adoc/trento-operations.adoc

```diff
@@ -0,0 +1,163 @@
+
+* Apply {saptune} Solution. Enabled when an {sap} workload has been discovered in the host but there is no {saptune} solution applied yet. {trento} restricts the solutions available for selection based on the nature of the {sap} workload. If the {sap} workload is a {hana} instance, the choice will be between HANA and S4HANA-DESERVER . If the {sap} workload is an application instance, the choice will be between NETWEAVER and S4HANA-APPSERVER.
+** Activity type: Host Operation Requested
+** Internal policies: the {sap} workload must be stopped
+** Internal timeout: 5m
+
+* Change {saptune} Solution. Enabled when an {sap} workload has been discovered in the host and there is already an applied {saptune} solution. Same as with the apply operation, {trento} will restrict the solutions available for selection based on the nature of the {sap} workload.
+** Activity type: Host Operation Requested
+** Internal policies: the {sap} workload must be stopped
+** Internal timeout: 5m
+
+* Reboot Host. It schedules a reboot of the target host within a minute of the request time:
+** Activity type: Host Operation Requested
+** Internal policies:
+*** If an {sap} workload has been discovered in the host, it must be stopped.
+*** If the host is a cluster node, the {pace} service must be disabled at boot and stopped (the node must be offline node).
+** Internal timeout: 5m
+
+=== Cluster operations
+
+Every time a user requests a cluster operation, {trento} checks the cluster state using `cs_clusterstate -i`. If the {tragent} cannot find the `cs_clusterstate` executable or it returns any state other than IDLE, the operation fails. This restriction has been implemented to ensure that a user cannot execute an operation while the cluster is in transition.
+
+NOTE: The `cs_clusterstate` executable is delivered with the `ClusterTools2` package.
+
+For any registered cluster, the following operations are available in the corresponding details view:
+
+* At cluster level:
+** Cluster maintenance. Enabled when at least one node in the cluster is online. It turns maintenance on or off in the cluster, depending on its current status.
+*** Activity type: Cluster Operation Requested
+*** Internal policies: not applicable
+*** Internal timeout: 5m
+
+* At node level:
+** Node maintenance. Enabled when the node is online. It turns maintenance on or off in the node, depending on its current status.
+*** Activity type: Operation Requested on a cluster host
+*** Internal policies: not applicable
+*** Internal timeout: 5m
+** Enable {pace} at boot: Enabled when the service is disabled at boot in the corresponding host.
+*** Activity type: Operation Requested on a cluster host
+*** Internal policies: not applicable
+*** Internal timeout: 5m
+** Disable {pace} at boot: Enabled when the service is enabled at boot in the corresponding host.
+*** Activity type: Operation Requested on a cluster host
+*** Internal policies: not applicable
+*** Internal timeout:: 5m
```

Comment:
There actually is an internal policy for these, which are the mentioned conditions about when the operation is enabled
- Enable pacemaker: the service is disabled at boot in the corresponding host
- Disable pacemaker: the service is enabled at boot in the corresponding host

I am fine with the content, just wanted to highlight that from an API perspective you could still request the operation, but the policy will take place.

From a UI perspective this is the button being enabled or not.